airbyte-integrations/connectors/source-close-com/acceptance-test-config.yml

```diff
@@ -14,10 +14,24 @@ tests:
   basic_read:
```

Comment:
could you please migrate SAT config to the new version?

Reply:
New version does not support `empty_streams`

Reply:
Done